core/App/screens/ContactDetails.tsx

```diff
@@ -154,7 +154,7 @@ const ContactDetails: React.FC<ContactDetailsProps> = ({ route }) => {
           {
             name: connection?.alias || connection?.theirLabel,
             value: t('ContactDetails.DateOfConnection', {
-              date: connection?.createdAt.toLocaleString(i18n.language, dateFormatOptions),
+              date: connection?.createdAt !== undefined ? formatTime(connection.createdAt) : '',
```

Comment:
Just FYI: The inclusion of the optional chaining (`?.`) operator in `connection?.createdAt` is the same thing as `connection.createdAt !== undefined`. So you can simply write `connection?.createdAt ? formatTime(connection.createdAt) : ''`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining